tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
```

Comment:
The present form was already present in the transation I worked upon ([here](https://github.com/Clonewayx/Tuxemon/blob/3a85ae69b7507d413ce97eaffaa14b6aa498d0e5/tuxemon/resources/db/locale/eo_EU.json)). I didn't check it, my fault. I fix it